swag.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+order: 0
+section: "2018"
+title: "Swag"
+titleDisplay: "Swag"
+parent: "Conference"
+redirect_from:
+  - "/2018/swag/"
+---
+
+What better way to celebrate Doing It With Others than by buying [t-shirts](#t-shirts), [totes](#totes), buttons and stickers?
+
+<br />
+
+## T-Shirts - $20
+<a id="t-shirts"></a>
+
+We have 4 shirt designs available for purchase during [registration](/conference/#registration) (also for sale at the conference). Available S-XL in straight cut and fitted options. Printed locally on Bella+Canvas [3001](https://www.bellacanvas.com/product/3001/Unisex-Jersey-Short-Sleeve-Tee.html) or [6004](https://www.bellacanvas.com/product/6004/Womens-The-Favorite-Tee.html).
+
+<div class="row">
+  <div class="four columns">
+    <h3>2018 T-Shirt Design #1</h3>
```

Comment:
Currently we have these names:

* 2018 T-Shirt Design #1
* 2018 T-Shirt Design #2
* T-Shirt Design Black
* T-Shirt Design White

![screen shot 2018-05-28 at 4 00 55 pm](https://user-images.githubusercontent.com/2261308/40628032-8869bab2-6290-11e8-8768-6d97cd1a5afc.png)

I suggest either:

* 2018 T-Shirt Light
* 2018 T-Shirt Dark
* T-Shirt Design Dark
* T-Shirt Design Light

Or:

* Design #1: 2018 T-Shirt
* Design #2: 2018 T-Shirt
* Design #3: T-Shirt
* Design #4: T-Shirt